src/commands/transaction/addressalias.ts

```diff
@@ -91,6 +101,11 @@ export default class extends ProfileCommand {
             throw new ExpectedError('You need to introduce namespace id.');
         }
 
+        options.maxFee = OptionsResolver(options,
+            'maxFee',
+            () => undefined,
+            'maxFee: ');
```

Comment:
'Introduce the maximum fee you want to spend to announce the transaction':

(applies to every file)

Reply:
Wait a while. I'll add the validator that didn't write the test before.https://github.com/nemtech/nem2-cli/issues/53